src/language/__tests__/printLocation-test.js

```diff
@@ -7,7 +7,48 @@ import dedent from '../../jsutils/dedent';
 import { Source } from '../../language';
 import { printSourceLocation } from '../printLocation';
 
-describe('printLocation', () => {
+describe('printSourceLocation', () => {
+  it('prints minified documents', () => {
+    const minifiedSource = new Source(
+      'query SomeMiniFiedQueryWithErrorInside($foo:String!=FIRST_ERROR_HERE$bar:String){someField(foo:$foo bar:$bar baz:SECOND_ERROR_HERE){fieldA fieldB{fieldC fieldD...on THIRD_ERROR_HERE}}}',
+    );
+
+    const firstLocation = printSourceLocation(minifiedSource, {
+      line: 1,
+      column: minifiedSource.body.indexOf('FIRST_ERROR_HERE') + 1,
+    });
+    expect(firstLocation + '\n').to.equal(dedent`
+      GraphQL request:1:53
+      1 | query SomeMiniFiedQueryWithErrorInside($foo:String!=FIRST_ERROR_HERE$bar:String)
+        |                                                     ^
+        | {someField(foo:$foo bar:$bar baz:SECOND_ERROR_HERE){fieldA fieldB{fieldC fieldD.
```

Comment:
> Queries, particularly when compressed, can have a long line length. In narrow viewports (small Terminal windows, etc.) this results in the query wrapping over multiple lines.

@jaydenseric I think I finally figured out a generic solution to the problem that you described in #1786. This test (+ 2 test cases below) shows the new format handles error location in the beggining, in the middle and in the end of the minified document.
What do you think?